src/database/populate.py

```
@@ -57,23 +59,27 @@ def _preprocess_csv(self) -> pd.DataFrame:
         :return: A Pandas DataFrame containing cleaned movie data.
         """
         data = pd.read_csv(self._csv_file_path)
-        data = data.drop_duplicates(subset=['names', 'date_x'], keep='first')
+        data = data.drop_duplicates(subset=["names", "date_x"], keep="first")
 
-        for col in ['crew', 'genre', 'country', 'orig_lang', 'status']:
-            data[col] = data[col].fillna('Unknown').astype(str)
+        for col in ["crew", "genre", "country", "orig_lang", "status"]:
+            data[col] = data[col].fillna("Unknown").astype(str)
 
-        data['crew'] = (
-            data['crew']
-            .str.replace(r'\s+', '', regex=True)
-            .apply(lambda x: ','.join(sorted(set(x.split(',')))) if x != 'Unknown' else x)
+        data["crew"] = (
+            data["crew"]
+            .str.replace(r"\s+", "", regex=True)
+            .apply(
+                lambda x: ",".join(sorted(set(x.split(",")))) if x != "Unknown" else x
+            )
         )
 
-        data['genre'] = data['genre'].str.replace('\u00A0', '', regex=True)
-        data['date_x'] = data['date_x'].astype(str).str.strip()
-        data['date_x'] = pd.to_datetime(data['date_x'], format='%Y-%m-%d', errors='raise')
-        data['date_x'] = data['date_x'].dt.date
-        data['orig_lang'] = data['orig_lang'].str.replace(r'\s+', '', regex=True)
-        data['status'] = data['status'].str.strip()
+        data["genre"] = data["genre"].str.replace("\u00a0", "", regex=True)
+        data["date_x"] = data["date_x"].astype(str).str.strip()
+        data["date_x"] = pd.to_datetime(
+            data["date_x"], format="%Y-%m-%d", errors="raise"
+        )
```

Robustness: `pd.to_datetime(..., errors='raise')` will raise and crash the seeder on any malformed or unexpected date string. If the CSV may contain stray formats, consider using `errors='coerce'` and handling `NaT` values (drop, log, or infer), or validate/transform problematic rows with a clear error message. Failing fast may be acceptable for controlled datasets, but it's brittle for real-world CSVs.